conf.py

```diff
@@ -20,7 +20,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "Multinet"
-copyright = "2019, Multinet Team"
+copyright = "2019-2020, Multinet Team"
```

Comment:
According to [this](https://www.copyrightlaws.com/copyright-symbol-notice-year/), the important thing is the year of first publication. I'm thinking we should leave this alone because it creates a maintenance burden (however slight) if we want to keep this updated to the current year.

```suggestion
copyright = "2019, Multinet Team"
```